resources/qml/delegates/MessageDelegate.qml

```diff
@@ -13,7 +13,7 @@ Item {
 
     required property bool isReply
     property alias child: chooser.child
-    property real implicitWidth: (chooser.child && chooser.child.implicitWidth) ? chooser.child.implicitWidth : width
+//    property real implicitWidth: (chooser.child && chooser.child.implicitWidth) ? chooser.child.implicitWidth : width
```

Comment:
This was used to make the reply background adapt to the size of the text messages. I would like to keep the reply backgrounds the same size as their message content.

Reply:
keeping it doesn't seem to cause any trouble